Makefile

```diff
@@ -69,38 +71,42 @@ test/JSON-Schema-Test-Suite/tests:
 .PHONY: test
 # Would be nice to include elvis to test, but it fails on OTP-18
 # test: elvis
-test: eunit ct xref dialyzer cover
+test: eunit ct xref dialyzer cover proper
```

Comment:
let's put `proper` before `cover`, so coverage generated by running `proper` is also included to cover report